test/blackbox-tests/test-cases/github4401.t

```diff
@@ -21,3 +21,12 @@ are ignored. See #4401.
   Error: No rule found for test
   -> required by alias runtest in dune:5
   [1]
+
+This is correctly ignored if dune-lang is bumped to 3.5.
```

Comment:
```suggestion
This is correctly ignored if `dune-lang` is bumped to 3.5.
```

Reply:
@christinerose This is inside a cram test however and not in any documentation. So monospace is really up to the font you are using 0:-)